projects/cashmere/src/lib/pipes/age-pipe/age.pipe.spec.ts

```diff
@@ -0,0 +1,105 @@
+
+import {async, inject, TestBed} from '@angular/core/testing';
+
+import { AgePipe } from './age.pipe';
+import { Date } from 'sugar';
+
+describe('AgePipe', () => {
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
```

Comment:
I would prefer to not use TestBed for testing Pipes, unless it is necessary to test integration with other Angular things.  You should just be able to `new()` up the pipe and test it.

Reply:
Fixed